.updatecli/bump-opbeans-node.yml

```diff
@@ -0,0 +1,65 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it.
+name: Bump elastic-apm-node to latest version
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: opbeans-node
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  opbeans-node:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+        - javascript
+
+
+sources:
+  elastic-apm-node:
+    name: Get latest elastic-apm-node version from npm registry
+    kind: npm
+    spec:
+      name: elastic-apm-node
+
+conditions:
+  elastic-apm-node-version-check:
+    name: Check if installed version differs
+    kind: shell
+    sourceid: elastic-apm-node
+    spec:
+      command: bash .updatecli/scripts/test-version.sh
+      environments:
+        - name: PATH
+
+targets:
+  dockerfile:
+    name: Set org.label-schema.version in Dockerfile
+    sourceid: elastic-apm-node
+    scmid: githubConfig
```

Comment:
@olblak `sourceid` and `scmid` are always the same when you only have one source and one scm. Could it make sense to put them to those values by default in case you do not pass `sourceid` and `scmid`? this will make the file less verbose,  In this case is not much but I have a file with more than 10 targets in other repo 😄